tools/debian/rules

```diff
@@ -18,7 +18,7 @@ override_dh_auto_configure:
 		--libexecdir=/usr/lib/cockpit $(CONFIG_OPTIONS)
 
 override_dh_auto_install:
-	dh_auto_install -- install-test-assets
+	dh_auto_install -- install-tests
```

Comment:
This is incomplete, as tools/debian/cockpit-tests.install still installs etc/systemd/system/cockpit.service.d. However, as I said above let's not do this dance: Let's kill tools/debian/cockpit-tests.install and instead in `override_dh_install:` do `make install-tests DESTDIR=debian/cockpit-tests/` instead. There is no point in duplicating the file list downstream.